src/mongocxx/collection.hpp

```diff
@@ -602,57 +562,51 @@ class MONGOCXX_API collection {
    private:
     friend class database;
 
-    MONGOCXX_PRIVATE collection(const database& database, bsoncxx::stdx::string_view collection_name);
+    MONGOCXX_PRIVATE collection(const database& database,
+                                bsoncxx::stdx::string_view collection_name);
+
+    MONGOCXX_PRIVATE collection(const database& database,
```

Comment:
This is a new constructor (just pointing out the non-clang-format change in this file)